### PR TITLE
Add event schedule block to event page

### PR DESCRIPTION
**Pull request by Siilwyn**

### CODE REVIEW

File: components/event-block/EventBlockSchedule.vue [hunk 1]
```diff
@@ -0,0 +1,539 @@
...
+        currentDate: new Date(),
+        selectedEventDayId: '',
+        displayTimezone: `${timezoneComment} UTC ${timezone.substr(0,3)}`,
+      };
+    },
+    computed: {
+      parsedEventDays({ eventDays, currentDate, timezone }) {
+        return eventDays.map(eventDay => ({
+          ...eventDay,
+          parsedDate: new Date(`${eventDay.date}T07:00${timezone}`),
```
Comment (Siilwyn) [inline components/event-block/EventBlockSchedule.vue:233]: To also shift the timezone a time is needed, not sure what to put in.

File: components/event-block/EventBlockSchedule.vue [hunk 2]
```diff
@@ -0,0 +1,539 @@
...
+          role="tab"
+          href="#"
+          :id="`tab-wide-${eventDay.id}`"
+          :aria-controls="`section-wide-${eventDay.id}`"
+          :aria-selected="eventDay.id === activeEventDayId"
+          @click.prevent="selectedEventDayId = eventDay.id"
+        >
+          <time :datetime="eventDay.date">
+            {{
+              new Date(eventDay.parsedDate).toLocaleDateString(
```
Comment (jbmoelker) [inline components/event-block/EventBlockSchedule.vue:21]: **suggestion:** use helper method `displayDate(eventDay.parsedDate)` so you can reuse it in mobile template?

File: components/event-block/EventBlockSchedule.vue [hunk 3]
```diff
@@ -0,0 +1,539 @@
...
+                  <span class="event-block-schedule__description-label">Topic</span>
+                  <span class="event-block-schedule__topic">{{ scheduleItem.topic }}</span>
+                </div>
+              </div>
+              <div class="event-block-schedule-wide__description">
+                <span class="event-block-schedule__description-label">Description</span>
+                {{ scheduleItem.description }}
+              </div>
+              <div class="event-block-schedule-wide__url-wrapper">
+                <a
```
Comment (jbmoelker) [inline components/event-block/EventBlockSchedule.vue:81]: `target="_blank" rel="noopener"`

File: components/event-block/EventBlockSchedule.vue [hunk 4]
```diff
@@ -0,0 +1,539 @@
...
+                { weekday: 'short', month: 'long', day: 'numeric' }
+              )
+            }}
+          </time>
+        </a>
+      </div>
+
+      <p>{{ displayTimezone }}</p>
+
+      <section
```
Comment (jbmoelker) [inline components/event-block/EventBlockSchedule.vue:32]: share partials between desktop and mobile template?

**Pull request merged**